<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="WTC-5159473-l2" name="WERL_Application_Test.SmokeTest" tests="1" failures="1" timestamp="23 Oct 2020 21:20:13 GMT" time="77.950" errors="0">
  <testcase name="WERL_Registration" time="77.950" classname="WERL_Application_Test.SmokeTest">
    <failure type="java.lang.AssertionError" message="Element is not present on page.  By.id: firstNameField">
      <![CDATA[java.lang.AssertionError: Element is not present on page.  By.id: firstNameField
at org.testng.Assert.fail(Assert.java:96)
at WERL_Application_Test.SmokeTest.WERL_Registration(SmokeTest.java:78)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- WERL_Registration -->
</testsuite> <!-- WERL_Application_Test.SmokeTest -->
